client/src/panels/panel_declarations/covid/covid_estimates.js

```diff
@@ -434,69 +311,20 @@ const ByMeasureTab = wrap_with_vote_stat_controls(
   }
 );
 
-const tab_content_configs = [
-  {
-    key: "summary",
-    levels: ["gov", "dept"],
-    label: text_maker("summary_tab_label"),
-    load_data: ({ subject, selected_year }) =>
-      (() => {
-        if (subject.level === "dept") {
-          return query_org_covid_summary({
-            org_id: String(subject.id),
-            fiscal_year: selected_year,
-          });
-        } else {
-          return query_gov_covid_summary({
-            fiscal_year: selected_year,
-          });
-        }
-      })().then((covid_summary) => _.get(covid_summary, "covid_estimates")),
-    TabContent: SummaryTab,
-  },
-  {
-    key: "department",
-    levels: ["gov"],
-    label: text_maker("by_department_tab_label"),
-    load_data: ({ selected_year }) =>
-      query_all_covid_estimates_by_measure_id({
-        fiscal_year: selected_year,
-      }).then((data) =>
-        roll_up_flat_measure_data_by_property(data, "org_id", "est_doc")
-      ),
-    TabContent: ByDepartmentTab,
-  },
-  {
-    key: "measure",
-    levels: ["gov", "dept"],
-    label: text_maker("by_measure_tab_label"),
-    load_data: ({ subject, selected_year }) =>
-      (() => {
-        if (subject.level === "dept") {
-          return query_org_covid_estimates_by_measure_id({
-            org_id: String(subject.id),
-            fiscal_year: selected_year,
-          });
-        } else {
-          return query_all_covid_estimates_by_measure_id({
-            fiscal_year: selected_year,
-          });
-        }
-      })().then((data) =>
-        roll_up_flat_measure_data_by_property(data, "measure_id", "est_doc")
-      ),
-    TabContent: ByMeasureTab,
-  },
-];
-
 class CovidEstimatesPanel extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       loading: true,
       summary_by_fiscal_year: null,
       selected_year: _.last(props.panel_args.years),
+      selected: [
+        text_maker(`covid_estimates_stat`),
+        text_maker(`covid_estimates_vote`),
+      ],
```

Comment:
See the other comment first, because it gives some useful general advice (but it overall a bit undercut by this comment).

Since only `SummaryTab` cares about this state (not needed across tabs like the year), it'd be easier to make `SummaryTab` a class component (or use hooks) and make it own this state directly. Right? Or am I missing something?